client/profile-wizard/steps/business-details/flows/selective-bundle/index.js

```diff
@@ -91,6 +91,7 @@ class BusinessDetails extends Component {
 				extensionInstallationOptions[ 'woocommerce-services' ],
 			install_mailchimp:
 				extensionInstallationOptions[ 'mailchimp-for-woocommerce' ],
+			install_mailpoet: extensionInstallationOptions.mailpoet,
```

Comment:
In retrospect, we probably should have mapped the `extensionInstallationOptions` directly to the event property names, but I suppose this would break historical tracks data if we updated now.

Reply:
Yeah I agree. I did consider we could improve this and keep the prop names but the effort felt out of scope.